ansible/roles/test/files/ptftests/sad_path.py

```diff
@@ -36,16 +36,17 @@ def revert(self):
 
 class SadPath(object):
     def __init__(self, oper_type, vm_list, portchannel_ports, vm_dut_map, test_args):
-        self.oper_type = oper_type
+        (self.oper_type, self.cnt) = oper_type.split(':') if ':' in oper_type else (oper_type, 1)
```

Comment:
>(self.oper_type, self.cnt) = oper_type.split(':') if ':' in oper_type else (oper_type, 1) [](start = 8, length = 89)

What is your design of the constructor parameter? Why not assume it in one format?

Reply:
oper_type will either be 'value' or 'value:cnt' if cnt is greater than 1. Kept it this way since I didn't want to make changes in parameter format for the existing single member case. 

I will have to pass the cnt as an additional parameter if I do the split outside the class.

Reply:
I mean change everything into 'value:cnt' will make code easier.

---
In reply to: [303570408](https://github.com/Azure/sonic-mgmt/pull/1004#discussion_r303570408) [](ancestors = 303570408)

Reply:
I will be adding  one more parameter for lag member down case. so it will be value:cnt1:cnt2 . cnt1 will be the same as the number of VMS. cnt2 will be number of lag members. 
I prefer to handle it in the code for cases that don't need it  rather than doing it in the yml.  for eg, 
single bgp member down should be changed to 'bgp_down:1:0' in the yml or I can keep it as 'bgp_down' in the yml and handle it in the script.